### PR TITLE
Voting UI

### PR DESCRIPTION
**Pull request by Alain1405**
https://github.com/raphaelmerx/imigrasi-nextjs/assets/2998545/9f2ca5ca-1986-4ed9-9b91-591fe837786d

### CODE REVIEW

**Comment by Alain1405** [comment]
@raphaelmerx it says deployment failed. Hopefully it doesn't deploy automatically from any branch?

**Comment by raphaelmerx** [comment]
No it wouldn't deploy to the prod URL, only to a side URL. In this instance it got a build failure: `Type error: Module '"@/utils"' has no exported member 'supabaseAdmin'.`

File: components/Answer/Answer.tsx [hunk 1]
```diff
@@ -1,18 +1,78 @@
...
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (!response.ok) {
+      /* Handle */
+      alert('Voting error');
+    } else {
+      setVotingVisible(false);
+      setFeedbackMsg('Thank you for your vote!');
```
Comment (raphaelmerx) [inline components/Answer/Answer.tsx:46]: Added a bit of user feedback here:

Screenshot 2023-05-26 at 09 00 39

File: README.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ Backend is at https://github.com/raphaelmerx/imigrasi-bot
 
 Setup:
 * Install deps: `npm install`
-* Create a `.env` file with a `NEXT_PUBLIC_BACKEND_URL` variable (prefix the url with `ws://` instead of `http://`)
+* Create a `.env` file with the backend variables, e.g.
+```
+NEXT_PUBLIC_BACKEND_URL="127.0.0.1:9000"
```
Comment (raphaelmerx) [inline README.md:9]: Standardised this to one URL, the frontend takes care of setting the scheme

**Review by raphaelmerx — APPROVED** [review]

**Pull request merged**